src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1493,7 +1492,7 @@ string[] linkingTargetFields
         /// <param name="pathContainsReservedCharacters">Boolean indicating if path prefix contains reserved characters.</param>
         /// <param name="expectError">Exception expected</param>
         /// <param name="apiType">Either REST or GraphQL</param>
-        private static void ValidateRestAndGraphQLPathIsWellFormed(
+        private static void ValidateRestAndGraphQLURIIsWellFormed(
```

Comment:
The many conditionals in this method seem to complicate what you are trying to test. Is there any cleaner way to accomplish the asserts on the error messages you are expecting? It looks like `ApiType` isn't necessary.

Reply:
Simplified!